arbor/profile/meter_manager.cpp

```diff
@@ -18,6 +17,13 @@ namespace profile {
 using timer_type = timer<>;
 using util::strprintf;
 
+template <typename C>
+typename util::sequence_traits<C>::value_type
+mean(C const& c)
+{
+    return util::sum(c)/std::size(c);
+}
+
```

Comment:
There was always a latent issue in `mean` where the mean of sequence over an integral type would give an integral result. In this context, in `meter_manager.cpp`, it's always assigned to a double value, and so I think it's worthwhile making it a bit more correct and a bit less general, with e.g. `template <typename C> double mean(const C& c) { return util::sum(c)/(double)std::size(c); }`